dropwizard-validation/src/test/resources/logback-test.xml

```diff
@@ -0,0 +1,11 @@
+<configuration>
+    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder>
+            <outputPatternAsHeader>false</outputPatternAsHeader>
```

Comment:
This feature is [documented](https://logback.qos.ch/manual/encoders.html#outputPatternAsHeader) as disabled by default. Any reason this line must be added?

Reply:
Probably copy & pasted from some other source. Feel free to remove this line.